modules/nw-proxy-configure-object.adoc

```diff
@@ -81,18 +81,20 @@ metadata:
   name: cluster
 spec:
   httpProxy: http://<username>:<pswd>@<ip>:<port> <1>
-  httpsProxy: https://<username>:<pswd>@<ip>:<port> <2>
+  httpsProxy: http://<username>:<pswd>@<ip>:<port> <2>
```

Comment:
@danehans Please verify that the changes to this file are appropriate in response to your comments on the install docs that use these similar examples and descriptions. It can be previewed [here](https://osdocs-640--ocpdocs.netlify.com/openshift-enterprise/latest/networking/enable-cluster-wide-proxy.html).